rust/hyperlane-core/src/types/mod.rs

```diff
@@ -19,6 +23,82 @@ mod message;
 /// Unified 32-byte identifier with convenience tooling for handling
 /// 20-byte ids (e.g ethereum addresses)
 pub mod identifiers;
+mod primitive_types;
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Copy, Hash)]
+/// An ECDSA signature
```

Comment:
nit: ditto can you add a comment mentioning this is borrowed from https://github.com/hyperlane-xyz/ethers-rs/blob/hyperlane/ethers-core/src/types/signature.rs#L54